src/Compilers/VisualBasic/Portable/Semantics/OverloadResolution.vb

```diff
@@ -227,6 +231,28 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
 
                 Return False
             End Function
+
+            Public Overrides ReadOnly Property OverloadResolutionPriority As Integer
+                Get
+                    Return m_Method.OriginalDefinition.OverloadResolutionPriority
```

Comment:
>OriginalDefinition [](http://example.com/codeflow?start=36&length=18)

Are we testing cases that require `OriginalDefinition` in `OverloadResolutionPriority` or `GetOverloadResolutionPriorityInfo()`?

Reply:
This is not required for correctness.